variables.tf

```diff
@@ -132,3 +132,9 @@ variable "ec2_worker_volume_size" {
   default     = 40
   description = "The size, in GB, of the root EBS volume for the EC2 workers."
 }
+
+variable "ec2_worker_ami_id" {
+  type        = string
+  default     = ""
```

Comment:
I'm going to look into whether this should be `null` or just not defined here. The latest versions of TF have made some improvements to default values.